session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -1978,14 +1985,172 @@ public void createSchemaTemplate(
       List<List<String>> measurements,
       List<List<TSDataType>> dataTypes,
       List<List<TSEncoding>> encodings,
-      List<CompressionType> compressors)
+      List<List<CompressionType>> compressors)
       throws IoTDBConnectionException, StatementExecutionException {
     TSCreateSchemaTemplateReq request =
         getTSCreateSchemaTemplateReq(
             name, schemaNames, measurements, dataTypes, encodings, compressors);
     defaultSessionConnection.createSchemaTemplate(request);
   }
 
+  /** New interface for no schemaNames */
```

Comment:
Remove this, for new users, they do not understand.
Besides, the template in 0.12.2 do not have much user

Reply:
deleted